test/ex_doc/formatter/html/templates_test.exs

```diff
@@ -242,9 +242,16 @@ defmodule ExDoc.Formatter.HTML.TemplatesTest do
           tasks: []
         })
 
-      assert content =~ ~r{<li><a id="modules-list-link" href="#full-list">Modules</a></li>}
-      refute content =~ ~r{<li><a id="exceptions-list" href="#full-list">Exceptions</a></li>}
```

Comment:
I am not sure what this is about. I didn't see "exceptions" as a possible tab. It's not in `SIDEBAR_NAV_TYPES`.

Reply:
It was a very old tab, it no longer exists. :)

Reply:
I'll remove this assertion then